male-seed-beetle/male_seed_beetle/package.py

```diff
@@ -0,0 +1,38 @@
+import seedcase_sprout.core as sp
+import os
+import pathlib
+
+os.environ["SPROUT_GLOBAL"] = "./male-seed-beetle"
+
+package_paths = sp.create_package_structure(path=sp.path_packages())
+
+properties = sp.PackageProperties(
+    name="male-seed-beetle",
+    title="Complex mitonuclear interactions and metabolic costs of mating in male seed beetles",
+    description="Data from the 2015 on metabolic rate, respiratory quotient, body weight and ejaculate weight data from seed beetles with different mitonuclear genotypes.",
+    contributors=[
+        sp.ContributorProperties(
+            title="Jane Doe Test",
+            email="Jane.Doe@test.com",
+            path="example.com/jamie_jones",
+            roles=["creator"],
+        )
+    ],
+    licenses=[
+        sp.LicenseProperties(
+            name="CCO_1.0",
+            path="https://creativecommons.org/publicdomain/zero/1.0/legalcode",
+            title="CCO 1.0 UNIVERSAL",
+        )
+    ],
```

Comment:
Because we are using this data from another source location (Zenodo), you should add the `sources` field to link to the DOI.

Reply:
Like this?

Reply:
I found it and have updated.